client/flutter/test/home_page_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:WHOFlutter/main.dart';
+
+void main() {
+
+  final TestWidgetsFlutterBinding binding =
+      TestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Home Page Renders Correctly', (WidgetTester tester) async {
+    // Increasing the default viewport size to avoid RenderFlex overflow error
+    await binding.setSurfaceSize(Size(800, 640));
+    await tester.pumpWidget(MyApp());
+    // Delay to load the text from Localization
+    await tester.pumpAndSettle(Duration(seconds: 1));
+    // Expects the WHO logo displayed on the top
+    expect(find.byType(Image), findsOneWidget);
+    expect(find.text('Protect yourself'), findsOneWidget);
```

Comment:
This tests are very fragile, because HomePage uses localized content. And since you are pumping `MyApp()`, which automatically determines locale, I'm expecting this test will fail if launched on a device with a French language for example.

Reply:
What I can do is, wrap the `HomePage` with `MaterialApp` and define the locale to be en.

Reply:
Exactly 👍

Reply:
@AyushBherwani1998 seems like you're still using `MyApp` her instead of using `MaterialApp`